src/index.tsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { Provider } from 'react-redux';
+import { store } from './state/Store';
 
 ReactDOM.render(
     <React.StrictMode>
-        <App />
+        <Provider store={store}>
```

Comment:
I can move it to App but I also plan on connecting Options to Redux and at this point it's needed up here.

Reply:
I don't see it explicitly recommending it in index, just putting it at the top of the component hierarchy, which there's no real difference between being in index and being in App, as long as we continue to enforce not changing index (which I plan to)

Reply:
I think the main idea is that the store is created / provided by the thing that renders App - if you run App in a browser, the store is created in and provided by index.tsx since that's the browser entry point; if you run App in a test environment (e.g. using jest with react-testing-utils), you provide the store in your test code.